response/file.go

```diff
@@ -0,0 +1,84 @@
+package response
+
+import (
+	"io/ioutil"
+	"net/http"
+	"os"
+	"strings"
+)
+
+type FileResponder struct {
+	filepath string
+
+	resp *Response
+	err  error
+}
+
+func newFileResponder(filepath string) *FileResponder {
+	return &FileResponder{
+		filepath: filepath,
+	}
+}
+
+func File(filepath string) *FileResponder {
```

Comment:
I think `response.File()` should return `Response` struct.
otherwise, we can't use the method like `Status`, `Cookie`.

Reply:
Because this is wrapper of response, I don’t want to add more responsibility onto response

Reply:
We still need the File to help user parse the file and set Content-Type automatically.
but the `response.File` should have the same behavior as `response.New`, otherwise it is really confusing.
For example.
If the user wants to response a 500 like this.
```
response.File('500.html').Status(500);
```
but `FileResponder` don't support Status method. That is not something I think the user will be expected.